docs/using_the_android_sdk_bom.md

```diff
@@ -0,0 +1,38 @@
+# Using the Android SDK BOM in your own projects
+
+A BOM (Bill of Materials) is used to lock the dependencies (direct and transitive) down to known compatible versions. Gradle itself does not offer such a mechanism but it is a well known and used concept in the Maven world (see [Dependency Management](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management)).
+In order to bring this functionality to Gradle the Android SDK relies on the [dependency management plugin](https://github.com/spring-gradle-plugins/dependency-management-plugin) provided by Spring. This plugin allows us to use a standard Maven BOM in a Gradle project.
+
+If your project depends on the AeroGear Android SDK you can make use of the same BOM to ensure that your dependencies are compatible with the SDK. All you need to do is to apply the plugin and reference the BOM.
+
+At the top of your gradle build file (but below `buildscripts`) add the plugin itself:
+
+```
+plugins {
+    id "io.spring.dependency-management" version "1.0.4.RELEASE"
+}
+```
+
+and then before your `dependencies` add the `dependencyManagement` section that refers to our BOM:
+
+```
+dependencyManagement {
+    imports {
+        mavenBom 'org.jboss.aerogear:aerogear-android-sdk-bom:<latest version>'
+    }
+}
+```
+
+In case your project contains multiple modules you need to apply this to all your subprojects. Make sure to apply the previous steps to your parent modules `build.gradle` and wrap the `dependencyManagement` section as follows:
+
+```
+subprojects {
+    apply plugin: 'io.spring.dependency-management'
+
+    dependencyManagement {
+        imports {
+            mavenBom 'org.jboss.aerogear:aerogear-android-sdk-bom:<latest version>'
+        }
+    }
+}
+```
```

Comment:
Missing example with dependency.

Reply:
Done, added a usage example for a package we will definitely use (okhttp)